diff -urNad openoffice.org-dictionaries-2.2.0~/dictionaries/en_GB/en_GB.dic openoffice.org-dictionaries-2.0.2.99.2.0.3rc5/dictionaries/en_GB/en_GB.dic
--- dictionaries/en_GB/en_GB.dic	2006-09-10 08:58:17.000000000 +0200
+++ dictionaries/en_GB/en_GB.dic	2006-09-10 09:06:25.000000000 +0200
@@ -12207,6 +12207,7 @@
 debauchery/13	1
 Debbie/1
 Debby
+Debian
 debenture/2	1
 debilitate/6	1
 debilitation/2	1
