diff -urNad openoffice.org-dictionaries-2.2.0~/dictionaries/en_US/en_US.dic openoffice.org-dictionaries-2.2.0/dictionaries/en_US/en_US.dic
--- dictionaries/en_US/en_US.dic	2006-09-10 08:58:25.000000000 +0200
+++ dictionaries/en_US/en_US.dic	2006-09-10 09:00:34.000000000 +0200
@@ -6167,6 +6167,11 @@
 blocky/R
 bloc/MS
 Bloemfontein/M
+blog/SM
+blogged
+blogger/SM
+blogging
+blogged
 bloke/SM
 Blomberg/M
 Blomquist/M
@@ -7954,9 +7959,11 @@
 Caedmon/M
 Caesar/MS
 caesura/SM
+cafe/MS
 café/MS
 cafeteria/SM
 caffeine/SM
+caffeinated
 caftan/SM
 caged/U
 Cage/M
@@ -14151,6 +14158,7 @@
 Debee/M
 debenture/MS
 Debera/M
+Debian/M
 debilitate/NGXSD
 debilitation/M
 debility/MS
@@ -17132,6 +17140,7 @@
 eavesdropping
 eavesdrop/S
 eave/SM
+eBay/M
 Eba/M
 Ebba/M
 ebb/DSG
@@ -21140,6 +21149,7 @@
 Foch/M
 foci's
 focused/AU
+focussed
 focuser/M
 focuses/A
 focus/SRDMBG
@@ -22014,7 +22024,7 @@
 FSLIC
 ft/C
 FTC
-FTP
+ftp
 fuchsia/MS
 Fuchs/M
 fucker/M!
@@ -22572,6 +22582,7 @@
 gaudily
 gaudiness/MS
 gaudy/PRST
+gauge/SM
 gaugeable
 gauger/M
 Gauguin/M
@@ -22780,6 +22791,7 @@
 gentlewoman/M
 gentlewomen/M
 gently
+Gentoo/M
 gentrification/M
 gentrify/NSDGX
 Gentry/M
@@ -26556,7 +26568,8 @@
 ht
 HTML
 Hts/M
-HTTP
+http
+https
 Huang/M
 huarache/SM
 hubba
@@ -28173,6 +28186,7 @@
 inlander/M
 inlay/RG
 inletting
+inline/D
 inly/G
 inmost
 Inna/M
@@ -28569,7 +28583,7 @@
 internecine
 internee/SM
 interne's
-Internet/M
+internet/M
 INTERNET/M
 internetwork
 internist/SM
@@ -29433,6 +29447,7 @@
 jaunt/MDGS
 jaunty/SRTP
 Javanese
+JavaScript/M
 Java/SM
 javelin/SDMG
 Javier/M
@@ -39596,6 +39611,7 @@
 Onida/M
 onion/GDM
 onionskin/MS
+online
 onlooker/MS
 onlooking
 only/TP
@@ -41407,6 +41423,7 @@
 paymaster/SM
 payment/ASM
 Payne/SM
+PayPal/M
 payoff/MS
 payola/MS
 payout/S
@@ -42231,6 +42248,8 @@
 Phineas/M
 Phip/M
 Phipps/M
+phisher/MS
+phishing
 phi/SM
 phlebitides
 phlebitis/M
@@ -43032,6 +43051,7 @@
 pock/GDMS
 pockmark/MDSG
 Pocono/MS
+podcast/MSRG
 podded
 podding
 podge/ZR
@@ -43581,6 +43601,7 @@
 powerhouse/MS
 powerlessness/SM
 powerless/YP
+PowerPoint/M
 Powers
 Powhatan/M
 pow/RZ
@@ -49270,6 +49291,7 @@
 seaminess/M
 seamlessness/M
 seamless/PY
+SeaMonkey/M
 seam/MNDRGS
 seams/I
 seamstress/MS
@@ -51069,6 +51091,7 @@
 slacken/DG
 slacker/M
 slackness/MS
+Slackware/M
 slack/SPGTZXYRDN
 Slade/M
 slagged
@@ -52008,7 +52031,9 @@
 Spain/M
 spake
 Spalding/M
-Spam/M
+spam/MS
+spammer/MS
+spamming
 spa/MS
 Span
 spandex/MS
@@ -53948,6 +53973,7 @@
 Sunbelt/M
 sunblock/S
 sunbonnet/MS
+Sunbird/M
 sunburn/GSMD
 sunburst/MS
 suncream
@@ -54251,6 +54277,7 @@
 susceptibility/IM
 susceptible/I
 Susette/M
+SUSE/M
 sushi/SM
 Susie/M
 Susi/M
@@ -56024,6 +56051,7 @@
 thumb/SMDG
 thumbtack/GMDS
 thump/RDMSG
+Thunderbird/M
 thunderbolt/MS
 thunderclap/SM
 thundercloud/SM
@@ -57858,6 +57886,7 @@
 Ubangi/M
 ubiquitous/YP
 ubiquity/S
+Ubuntu/M
 Ucayali/M
 Uccello/M
 UCLA/M
@@ -58371,6 +58400,8 @@
 uninhibited/YP
 uninominal
 uninsured
+uninstall/GSMB
+uninstaller/MS
 unintellectual
 unintended
 uninteresting
@@ -58574,6 +58605,7 @@
 unsubdued
 unsubstantial
 unsubtle
+unsubscribe/D
 unsuitable
 unsuspecting/Y
 unswerving/Y
@@ -60432,6 +60464,8 @@
 webfeet
 webfoot/M
 Web/MR
+webmaster/MS
+webmistress/MS
 website/S
 web/SMR
 Webster/MS
